controllers/app.controllers.js

```diff
@@ -46,6 +47,29 @@ exports.getArticlesById = (req, res, next) => {
     });
 };
 
+exports.patchArticlesVotes = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  if (
+    inc_votes === undefined ||
+    typeof inc_votes !== "number" ||
+    isNaN(inc_votes)
+  ) {
+    const err = new Error("Bad request: invalid format");
+    err.status = 400;
+    return next(err);
+  }
```

Comment:
This type of error handling should be done in the model instead of the controller